contactsummary/js/src/contact-summary/services/contract.js

```diff
@@ -89,13 +89,31 @@ define([
 
     /**
      * @ngdoc method
-     * @name getContracts
+     * @name resetContracts
      * @methodOf ContractService
      * @returns {*}
      */
+    factory.resetContracts = function () {
+      contracts=[];
+      factory.collection = {
```

Comment:
Structure was getting initialized on top, I had copied the same.Code changed now to remove duplicasy